RAINCATCH-1172 docs changes

austincunningham opened the pull request:
# What:
Changes to RAINCATCH-1172 made by docs

# Why:
see if we can get tech approval for these changes.

jstaffor committed:
First draft after tech review - best to build this and review in browser before looking at changes

jstaffor committed:
First draft after tech review - best to build this and review in browser before looking at changes V2

jstaffor committed:
Attempt to sync docs with updates after tech-ack label

jstaffor committed:
Penultimate review - 2 comments require addressing before doc approval

jstaffor committed:
Update introducing-raincatcher-security-features.adoc

jstaffor committed:
Ready for final tech-ack

austincunningham committed:
resolving merge conflicts

jstaffor committed:
Major update after our conversation yesterday

austincunningham committed:
Merge branch 'Raincatch-1172' of github.com:austincunningham/raincatcher-docs into Raincatch-1172

austincunningham committed:
fixing links

austincunningham committed:
updates for links

austincunningham committed:
fixing broken links

austincunningham committed:
fixing broken links

austincunningham committed:
remove unwanted file from commit

witmicko commented on docs/workforce-management-framework/rhmap/master.adoc.orig:
```diff
@@ -0,0 +1,41 @@
+== Introducing {WFM-RC-NameLong}
+include::introducing-raincatcher.adoc[leveloffset=+2]
+
+<<<<<<< HEAD
```
git merge conflict

witmicko commented on docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc:
```diff
@@ -18,4 +18,6 @@ The following features are on the {WFM-RC-NameShort} roadmap:
 * Integration of sensor data
 
-//Features included in {WFM-RC-NameShort} {WFM-RC-Version} are:
+Features included in {WFM-RC-NameShort} {WFM-RC-Version} are:
```
maybe - '{WFM-RC-NameShort} {WFM-RC-Version} features: ' ?

jstaffor replied:
+1

witmicko reviewed (dismissed):
minor comments, looks good.
Can you add `*.orig` files to gitignore, or are they needed?

jstaffor committed:
Update con-introducing-raincatcher.adoc

jstaffor committed:
Docs-Ready

jstaffor committed:
master.adoc.orig file deleted

jstaffor commented:
@witmicko , @austincunningham  - if you can focus on this PR with the goal being that we get this merged by tomorrow COB - it would be a great achievement by all involved!!!

I've looked at it so many times now I cannot see "Can't see the wood for the trees". If there is doc related issues/suggestions - ping me and I'll make the changes after tech approval :+1:

witmicko commented:
> merged by tomorrow COB

sounds like a solid plan.

witmicko committed:
couple suggestions (#2)

Tech Approval and Doc Approval met - cellllllleeeeeeebrate good times, COME ON!!!!

* couple suggestions

* small fix

* small fixes

* Update con-passportauth-module.adoc

witmicko reviewed (approved):
looks good 👍
